### DISCUSSION TITLE
Pressure control setting not working.

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#102] [Q&A]
When I increase the pressure control setting to 100% I was expecting that if I press hard with my Apple Pencil I would the size or opacity to 100%(if I set size to 8px, pressing hard will make size 8 px) but it does not work: it only probably goes to 60%(if you set size to 8px pressing very hard on Pencil would only make the brush to 5px). The reason I say 60% is because when I use my finger it goes to 100% (much bigger or more opacity when using my finger).  Is this a bug.

**Comment by keiwando** [comment]
I have not been able to reproduce this issue. Can you maybe post a screen recording showing the problem and also showing all of the other brush settings?
If I create a new brush with all of the default settings and just drag the "Size > Pressure Control" slider to 100% I can reach the full size of the brush by pressing harder.
Here's an example where I set the brush size to 8px like you said, tapped the screen with my finger on the left first to get a reference for how big the full 8px size will be and then drew a line with Apple Pencil (middle) and then drew a line with my finger (right) for reference again.

![IMG_9176F8915125-1](https://user-images.githubusercontent.com/12989570/209529953-490676d0-8d72-414f-aff8-88347e30360f.jpeg)
